samcli/commands/deploy/auth_utils.py

```diff
@@ -101,6 +101,11 @@ def _auth_id(resources_dict, event_properties, identifier):
     """
     resource_name = event_properties.get(identifier, "")
     api_resource = resources_dict.get(resource_name, {})
+
+    # Auth does not apply to ApiGateway::RestApi resources so return true and continue
+    if api_resource and api_resource["Type"] == "AWS::ApiGateway::RestApi":
```

Comment:
Does this apply to [`AWS::ApiGatewayV2::Api`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-api.html) as well?

Reply:
Ah yea good catch. I think it should since if the event refers to an httpApi then it would have to be this